src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/help.py

```diff
@@ -9,93 +9,117 @@
+            short-summary: Manage elastic pools. An elastic pool is an allocation of CPU, IO, and memory resources. Databases inside the pool share these resources.
             """
-helps['sql db create'] = """
+helps['sql elastic-pool create'] = """
             type: command
-            short-summary: Create an Azure SQL database
+            short-summary: Creates an elastic pool
             """
-helps['sql db replication-link'] = """
+helps['sql elastic-pool update'] = """
+            type: command
+            short-summary: Updates an elastic pool
+            """
+#helps['sql elastic-pool db'] = """
+#            type: group
+#            short-summary: Command to manage databases activities in database elastic pools
+#            """
+#helps['sql elastic-pool recommended'] = """
+#            type: group
+#            short-summary: Commands to see information about an Azure SQL Recommended Elastic Pool
+#            """
+#helps['sql elastic-pool recommended db'] = """
+#            type: group
+#            short-summary: Commands to see information about an Azure SQL Database inside of Recommended Elastic Pool
+#            """
+helps['sql server'] = """
             type: group
-            short-summary: Manage database replication links
+            short-summary: Commands to manage servers.
             """
-helps['sql db data-warehouse'] = """
-            type: group
-            short-summary: Manage database data warehouse
+helps['sql server create'] = """
+            type: command
+            short-summary: Create a new Azure SQL server
             """
-helps['sql db restore-point'] = """
-            type: group
-            short-summary: Manage database restore points
+helps['sql server list'] = """
+            type: command
+            short-summary: List the SQL servers belong to given resource group.
             """
-helps['sql db transparent-data-encryption'] = """
-            type: group
-            short-summary: Manage database transparent data encryption
+helps['sql server update'] = """
+            type: command
+            short-summary: Update an Azure SQL server.
             """
-helps['sql db service-tier-advisor'] = """
+helps['sql server firewall-rule'] = """
             type: group
-            short-summary: Manage database service tier advisors
+            short-summary: Commands to manage Azure SQL server's firewall rules
             """
-helps['sql elastic-pools'] = """
-            type: group
-            short-summary: Commands to manage database elastic pools
+#helps['sql server firewall-rule allow-all-azure-ips'] = """
+#            type: command
+#            short-summary: Create a firewall rule that allows all Azure IP addresses to access the server
+#            """
+helps['sql server firewall-rule create'] = """
+            type: command
+            short-summary: Create a firewall rule that allows an IP address range to access the server
```

Comment:
I'm explaining 0.0.0.0 in the Start/End param, the description from these is directly pulled from swagger. https://github.com/Azure/azure-rest-api-specs/pull/954/commits/911dae4e7eb78669a784c9c830ae265c42e3c366